lib/rouge/lexers/javascript.rb

```diff
@@ -98,26 +98,23 @@ def self.detect?(text)
 
       def self.keywords
         @keywords ||= Set.new %w(
-          for in of while do break return continue switch case default
-          if else throw try catch finally new delete typeof instanceof
-          void this yield import export from as async super this
+          as async await break case catch continue debugger default delete
+          do else export finally from for if import ininstanceof new of
```

Comment:
There's a typo here: `in` and `instanceof` should have a space between them.

Reply:
Thank you for pointing that out. fixed it.